SeafShare/SeafShareAccountViewController.m

```diff
@@ -0,0 +1,99 @@
+//
+//  SeafShareAccountViewController.m
+//  seafilePro
+//
+//  Created by three on 2018/8/2.
+//  Copyright © 2018年 Seafile. All rights reserved.
+//
+
+#import "SeafShareAccountViewController.h"
+#import "SeafGlobal.h"
+#import "SeafAccountCell.h"
+#import "Debug.h"
+#import "UIViewController+Extend.h"
+
+@interface SeafShareAccountViewController ()<UITableViewDataSource, UITableViewDelegate>
+
+@property (nonatomic, strong) UITableView *tableView;
+
+@end
+
+@implementation SeafShareAccountViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.navigationItem.title = NSLocalizedString(@"Accounts", @"Seafile");
+    
+    [SeafGlobal.sharedObject loadAccounts];
+    
+    self.tableView = [[UITableView alloc] initWithFrame:self.view.bounds style:UITableViewStylePlain];
+    self.tableView.rowHeight = 64;
+    self.tableView.tableFooterView = [UIView new];
+    self.tableView.separatorStyle = UITableViewCellSelectionStyleNone;
+    self.tableView.delegate = self;
+    self.tableView.dataSource = self;
+    [self.view addSubview:self.tableView];
+}
+
+#pragma mark - Table view
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+    return 1;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    return SeafGlobal.sharedObject.conns.count;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
```

Comment:
this code block has been used a few times, maybe we should move it into SeafAccountCell.m, then it could be shared